apps/web-remix/app/routes/forms.$organizationId.pipelines.$pipelineId.tsx

```
@@ -53,144 +152,419 @@ export default function WebsiteForm() {
-      formData.append(`wait_for_outputs[${index}][block_name]`, output.name);
-      formData.append(`wait_for_outputs[${index}][output_name]`, 'output');
+    inputs.forEach((input) => {
+      push(`${input.name}:${input.input}`, input.value);
     });
+  };
+
+  const { onSubmit, ...formProps } = useForm({
+    onSubmit: handleOnSubmit,
+    defaultValues: pipeline.interface_config.form.inputs.reduce(
+      (acc, curr) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)